Makefile

```diff
@@ -122,17 +120,24 @@ all: $(BINDIR)/spread2d_test \
 	$(BINDIR)/cufinufft2d1many_test_32 \
 	$(BINDIR)/cufinufft2d2many_test_32 \
 	$(BINDIR)/cufinufft2d1nupts_test_32 \
-	$(BINDIR)/spread3d_test \
-	$(BINDIR)/spread3d_test_32 \
-	$(BINDIR)/interp3d_test \
-	$(BINDIR)/interp3d_test_32 \
 	$(BINDIR)/cufinufft3d1_test \
 	$(BINDIR)/cufinufft3d2_test \
 	$(BINDIR)/cufinufft3d1_test_32 \
 	$(BINDIR)/cufinufft3d2_test_32 \
 	$(BINDIR)/cufinufft2d2api_test \
 	$(BINDIR)/cufinufft2d2api_test_32 \
-	lib
+	lib spreadtest
+
+spreadtest: $(BINDIR)/spread2d_test \
+	$(BINDIR)/spread2d_test_32 \
+	$(BINDIR)/interp2d_test \
+	$(BINDIR)/interp2d_test_32 \
+	$(BINDIR)/spread3d_test \
+	$(BINDIR)/spread3d_test_32 \
+	$(BINDIR)/interp3d_test \
+	$(BINDIR)/interp3d_test_32
+	@echo "Running spread/interp only tests..."
+	(cd test; ./spreadperf.sh)
```

Comment:
There were some spread/interp only tests in `check` section. With this change, I think we can remove them there.

Reply:
I left these in, but rationalized all targets in a better way.